src/battle.ts

```diff
@@ -3546,23 +3546,24 @@ export class Battle {
 		}
 		case 'poke': {
 			let pokemon = this.rememberTeamPreviewPokemon(args[1], args[2])!;
-			if (args[3]) {
-				if (args[3] === 'mail') {
-					pokemon.item = '(mail)';
-				} else if (args[3] === 'item') {
-					pokemon.item = '(exists)';
-				} else if (this.rules['Tera Type Preview'] && args[3].startsWith('teraType:')) {
-					pokemon.teraType = args[3].slice(9);
-				} else if (this.rules['Force Open Team Sheets']) {
-					const setNotes = args[3].split(',');
-					[pokemon.name, pokemon.item, pokemon.ability, pokemon.teraType] = setNotes.slice(0, -1);
-					pokemon.ident = `${pokemon.side.sideid}: ${pokemon.name}`;
-					pokemon.searchid = `${pokemon.ident}|${pokemon.details}`;
-					pokemon.baseAbility = pokemon.ability;
-					for (const move of setNotes[setNotes.length - 1].split(';')) {
-						pokemon.rememberMove(move, 0);
-					}
+			if (args[3] === 'mail') {
+				pokemon.item = '(mail)';
+			} else if (args[3] === 'item') {
+				pokemon.item = '(exists)';
+			}
+			if (this.rules['Force Open Team Sheets']) {
+				pokemon.name = kwArgs.name;
+				pokemon.ident = `${pokemon.side.sideid}: ${pokemon.name}`;
+				pokemon.searchid = `${pokemon.ident}|${pokemon.details}`;
+				pokemon.item = kwArgs.item;
```

Comment:
In addition to ``item``, I will trouble you to investigate which of these need ``escapeHTML``. Likely, the answer is "all of them".

Tip: Most (if not all) of effect types in dex-data.ts have their .name property escaped.

Reply:
From what I could tell, `name` is stored as is and is escaped later for display purposes, such as in `showPokemonTooltip` of `battle-tooltips.ts`. The rest definitely need to be cleaned.